server/graphql/v1/inputTypes.js

```diff
@@ -485,7 +485,6 @@ export const ExpenseInputType = new GraphQLInputObjectType({
         deprecationReason: '2020-21-01: Please use PayoutMethod',
       },
       PayoutMethod: {
-        id: { type: GraphQLInt },
```

Comment:
@kewitz this was added in https://github.com/opencollective/opencollective-api/commit/a4693da584c86dda36d10bfdd95cc4f915cdb957 but I assume it has no effect because this is not a GraphQLObject, the fields are defined in `PayoutMethodInputType` not here. Can you confirm if it was just a typo?

Reply:
@betree, this was not a typo.
I temporarily had to add support for PayoutMethod in the current Expense flow, so we can select our TransferWise payout method when submitting expenses.

Reply:
But the input fields were already defined in `PayoutMethodInputType`, I don't understand why we needed to add an `id` here and how it influences the schema.

Reply:
Ah, now I get it. Yes, my mistake!